vpr/src/noc/noc_traffic_flows.cpp

```diff
@@ -44,10 +44,14 @@ std::vector<NocLinkId>& NocTrafficFlows::get_mutable_traffic_flow_route(NocTraff
     return traffic_flow_routes[traffic_flow_id];
 }
 
-const std::unordered_set<ClusterBlockId>& NocTrafficFlows::get_router_clusters_in_netlist(void) const {
+const std::vector<ClusterBlockId>& NocTrafficFlows::get_router_clusters_in_netlist(void) const {
```

Comment:
Can use .find () (and check if !std::end() and dereference). Can also use [] operator *if* you are sure the element is in the unordered_set.  If the element isn't there, it will be created, so don't use it if there's any doubt.

Reply:
Disregard -- didn't realize that this is a dense vector (entries [0 .. num_logical_routers-1]